android build error "Activity cannot be converted to Godot"

hello
when i try to build my project for android i receive an error

 ```
Task :compileDebugJavaWithJavac FAILED
project/android/fmod/src/FmodSingleton.java:32: error: incompatible types: Activity cannot be converted to Godot

```
i had to add implementation instead of compile() and make my build.gradle immutable or else it would change the file back and tell me that there was no compile() methods.

--- a/.github/workflows/all_builds.yml
+++ b/.github/workflows/all_builds.yml
@@ -272,7 +272,7 @@ jobs:
 
       - name: Compilation
         env:
-          ANDROID_NDK_ROOT: /usr/local/lib/android/sdk/ndk-bundle
+          ANDROID_NDK_ROOT: /usr/local/lib/android/sdk/ndk/21.4.7075529
         run: |
           cp -r godot-cpp ../godot-cpp
           cd ../godot-cpp

--- a/demo/android/fmod/src/FmodSingleton.java
+++ b/demo/android/fmod/src/FmodSingleton.java
@@ -13,7 +13,6 @@ public class FmodSingleton extends Godot.SingletonBase {
 
     protected Activity appActivity;
     protected Context appContext;
-    private Godot activity = null;
 
     static
     {
@@ -29,13 +28,7 @@ public class FmodSingleton extends Godot.SingletonBase {
         this.appContext = appActivity.getApplicationContext();
         // You might want to try initializing your singleton here, but android
         // threads are weird and this runs in another thread, so to interact with Godot you usually have to do.
-        this.activity = (Godot)p_activity;
         FMOD.init(p_activity);
-        this.activity.runOnUiThread(new Runnable() {
-            public void run() {
-                System.out.println("Fmod jar init");
-            }
-        });
     }
 
     // Forwarded callbacks you can reimplement, as SDKs often need them.

